libs/shared/flagd-js-core/src/lib/storage.ts

```diff
@@ -0,0 +1,27 @@
+import { FeatureFlag } from './feature-flag';
+import { parse } from './parser';
+
+/**
+ * The simple contract of the core logics storage layer.
+ */
+export interface Storage {
+  setConfigurations(cfg: string): void;
+
+  getFlag(key: string): FeatureFlag | undefined;
+}
+
+export class StorageImpl implements Storage {
```

Comment:
```suggestion
export class MemoryStorage implements Storage {
```

A bit of a nit, but I tend to dislike `Impl` names. I think `MemoryStorage` is a fair name, but there's probably others too, like `MapStorage` or something.

Reply:
Done